core/src/Cabana_Parallel.hpp

```diff
@@ -20,33 +20,27 @@
 #include <cstdlib>
 #include <type_traits>
 
-namespace Cabana
-{
+namespace Cabana {
```

Comment:
This is probably the main thing I don't like - having an open `{` or `(` not start on the next line.

Reply:
Fixed!